spacemesh/v2alpha1/tx.proto

```diff
@@ -2,14 +2,36 @@ syntax = "proto3";
 
 import "google/rpc/status.proto";
 
-package spacemesh.v2;
+package spacemesh.v2alpha1;
 
 message Transaction {
   oneof versioned {
     TransactionV1 v1 = 1;
   }
 }
 
+message ContentsSend {
+  string destination = 1; // recipient account
+  uint64 amount = 2; // amount in smidge
+}
+
+message ContentsSingleSigSpawn {
+  string pubkey = 1;
+}
+
+message ContentsMultiSigSpawn {
+  uint32 required = 1; // number of required signatures
+  repeated string pubkey = 2;
+}
+
+message TransactionContents {
```

Comment:
This should also be versioned, but it lives inside a versioned tx type so maybe that's enough. What do you think?